cloud-data-protection/src/components/logout.tsx

```diff
@@ -0,0 +1,26 @@
+import React, {useEffect} from "react";
+import {AuthService} from "../services/authService";
+import {useSnackbar} from 'notistack';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
import {useSnackbar} from "notistack";
```